.travis.yml

```diff
@@ -1,24 +1,19 @@
 language: python
+python:
+  - "3.5"
 install:
   - pip install coveralls tox
 script:
   - tox
 env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
   - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
   - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
   - TOXENV=py33-django17
-  - TOXENV=py33-django18
   - TOXENV=py34-django17
+  - TOXENV=py33-django18
```

Comment:
The order is a bit mixed up here. Can you sort the list by Python version first, Django version next?